_fluo-1-2/administration/run-fluo-in-docker.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Run Fluo in Docker
+category: administration
+order: 4
+---
+
+## Obtain the Docker image
+
+To obtain the docker image created by this project, you can either pull it from DockerHub at
+`apache/fluo` or build it yourself. To pull the image from DockerHub, run the command below:
+
+    docker pull apache/fluo
+
+While it is easier to pull from DockerHub, the image will default to the software versions below:
+
+| Software    | Version |
+|-------------|---------|
+| [Fluo]      | 1.2.0   |
+| [Accumulo]  | 1.8.1   |
+| [Hadoop]    | 2.7.3   |
+| [Zookeeper] | 3.4.9   |
+
+If these versions do not match what is running on your cluster, you should consider building
+your own image with matching versions. However, Fluo must be 1.2+. Below are instructions for
+building an image:
+
+1. Clone the Fluo docker repo
```

Comment:
We could put the build instructions in the projects readme and reference them from here.   This document would only have the instructions for running.

Reply:
Good idea. I will do that.